static/css/components/modal.less

```diff
@@ -0,0 +1,38 @@
+/* The Modal (background) */
+.modal {
```

Comment:
Don't we have an existing modal we can use?
e.g. the one for adding book-covers or creating a new list?

Reply:
Moving our modal to its own component file seems like a reasonable idea, but I don't think we want to create a new modal.

@jdlrobson thoughts/tips on how you'd like us to approach this?

Reply:
Don't know about that 🤔

Reply:
The modal we have is somewhat broken 
![Screenshot from 2019-03-19 12-34-47](https://user-images.githubusercontent.com/31198893/54586539-62102400-4a43-11e9-8572-1988ebb43dc8.png)
I have implemented something like this 
![Screenshot from 2019-03-16 02-11-43](https://user-images.githubusercontent.com/31198893/54586569-78b67b00-4a43-11e9-82ae-3059d28a63c0.png)

Reply:
$.colorbox is the modal we use. I think we should continue to use it for the time being to build lightboxes (however we might want to consider redesigning them and improving the CSS! :-)

The close icon spilling out of the modal is fixed in https://github.com/internetarchive/openlibrary/pull/1997

See http://www.jacklmoore.com/colorbox/